db/migrate/20180225105236_add_type_to_notifications.rb

```diff
@@ -0,0 +1,15 @@
+class AddTypeToNotifications < ActiveRecord::Migration[5.1]
+  def up
+    add_column :notifications, :type, :string
+    Notification.where(activity_type: 'Mention').update_all(type: 'mention')
```

Comment:
These are going to take a long time. I think there is an `in_batches` variant so it doesn't lock writes to the table as much